filament/src/RenderPass.h

```diff
@@ -234,21 +234,22 @@ class RenderPass {
         backend::Handle<backend::HwBufferObject> skinningHandle;        // 4 bytes
         backend::Handle<backend::HwBufferObject> morphWeightBuffer;     // 4 bytes
         backend::Handle<backend::HwSamplerGroup> morphTargetBuffer;     // 4 bytes
+        backend::Handle<backend::HwBufferObject> instanceBufferHandle;  // 4 bytes
```

Comment:
I'm starting to think that we might want to have an indirection here to store all three `Handle` related to skinning/morphing (it's going to be four soon).

Reply:
Yeah that would be nice. Let's do it in a separate PR?

Reply:
definitely, and in fact let's do this after the >4 bones PR.